temporian/implementation/numpy/data/event.py

```diff
@@ -94,6 +94,13 @@ def feature_names(self) -> List[str]:
         # indexes should have the same names
         return [feature.name for feature in self._first_index_features]
 
+    @property
+    def dtypes(self) -> Dict[str, type]:
```

Comment:
Nit: Functions that are only for the implementation can be made private using an "_" prefix.

Reply:
this might be useful for the end user too I think